app/assets/stylesheets/custom_css/event.css.scss

```diff
@@ -0,0 +1,231 @@
+.event {
+  
+  .navbar {
+    background-color: rgba(0,0,0,0.3);
+    height: 100px;
+    border: none;
+    padding-right: 10px;
+    position: fixed;
+    right: 0;
+    top: 0;
+    z-index: 10000;
+
+    li a {
+      color: #ffcb05;
+      font-size: 16px;
+      padding-top: 30px;
+    }
+
+    li a:visited {
+      color: #ffcb05;
+    }
+
+    li a:hover {
+      color: white;
+    }
+  }
+
+  /* the menu icon */
+  .menu_button {
+    display: none;
+
+    button {
+      position: fixed;
+      z-index: 10000;
+      top: 5px;
+      right: 5px;
+      width: 60px;
+      height: 60px;
+
+      span {
+        display: fixed;
+      }
+    }
+
+    .dropdown-menu {
+      position: fixed;
+      z-index: 10000;
+      top: 70px;
+      width: 100%;
+      float: none;
+
+      li a {
+        font-size: 16px;
+        border-top: 1px solid whitesmoke;
+        padding-bottom: 15px;
+        padding-top: 15px;
+        color: darkgrey;
+      }
+
+      li a:first-child {
+        border-top: none;
+      }
+    }
+  }
+
+  .main_cover {
+    height: 500px;
+    background: url('https://theadventuresofwhuffler.files.wordpress.com/2011/06/hong-kong-skyline1.jpg');
```

Comment:
you should include images from your assets folder, not directly from the internet